regexec.c

```diff
@@ -3699,7 +3699,7 @@ Perl_regexec_flags(pTHX_ REGEXP * const rx, char *stringarg, char *strend,
 
     /* Be paranoid... */
     if (prog == NULL) {
-        Perl_croak(aTHX_ "NULL regexp parameter");
+        croak("NULL regexp parameter");
```

Comment:
This should be a panic.

Reply:
That sounds like a good suggestion for another branch. In this branch I'm aiming not to have any behavioral changes.